README.md

```diff
@@ -153,6 +153,13 @@ usage is clear you can use `U`, `V` etc. for any following generic.
 If the usage is not obvious, you should use a more descriptive name. The same naming rules as for
 classes will apply then.
 
+### Prefixing
+
+#### Types and Interfaces
+
+Do not prefix types with a `T` or interfaces with an `I`. Make sure to name them with a descriptive
```

Comment:
Perhaps **Make sure to give them a descriptive name** ?

Reply:
Yes, changing ;)